platform-modules/manager/src/main/java/org/ekstep/taxonomy/mgr/impl/ContentManagerImpl.java

```diff
@@ -86,7 +86,8 @@
 import java.util.Optional;
```

Comment:
ContentManagerImpl.java - This file has more than 2k lines of code. For all new APIs or enhancements we updating the same file. Can you please re-design and split this code into multiple files.

Reply:
I will go through the implementation and will plan for splitting the implementations.